WireMessaging/Sources/WireMessagingDomain/WireCells/UseCases/UploadDraftUseCase.swift

```diff
@@ -139,4 +156,8 @@ package struct UploadDraftUseCase: WireCellsUploadDraftUseCaseProtocol, WireCell
         }
     }
 
+    private static func intermediaryFilesDirectory() -> URL {
+        URL.temporaryDirectory.appendingPathComponent(UUID().uuidString, isDirectory: true)
```

Comment:
question: so temp directory is wiped out at each launch, could this be an issue?